t/matrixSslLoadRsaKeys.t

```diff
@@ -36,14 +36,14 @@ is PS_SUCCESS, _load_rsa_mem(undef, undef, $trustedCA),
 is PS_SUCCESS, _load_rsa_mem($cert, $priv, $trustedCA),
     'Mem both';
 
-is PS_SUCCESS, _load_rsa($certFile, $privFile_des3, $privPass_des3, undef),
-    'server: encrypted des3, RIGHT PASSWORD';
-is PS_ARG_FAIL, _load_rsa($certFile, $privFile_des3, $privPass, undef),
-    'server: encrypted des3, NO PASSWORD';
-is PS_FAILURE, _load_rsa($certFile, $privFile_des3, '', undef),
-    'server: encrypted des3, EMPTY PASSWORD';
-is PS_FAILURE, _load_rsa($certFile, $privFile_des3, 'WrOnG', undef),
-    'server: encrypted des3, WRONG PASSWORD';
+#is PS_SUCCESS, _load_rsa($certFile, $privFile_des3, $privPass_des3, undef),
+#    'server: encrypted des3, RIGHT PASSWORD';
+#is PS_ARG_FAIL, _load_rsa($certFile, $privFile_des3, $privPass, undef),
+#    'server: encrypted des3, NO PASSWORD';
+#is PS_FAILURE, _load_rsa($certFile, $privFile_des3, '', undef),
+#    'server: encrypted des3, EMPTY PASSWORD';
+#is PS_FAILURE, _load_rsa($certFile, $privFile_des3, 'WrOnG', undef),
+#    'server: encrypted des3, WRONG PASSWORD';
```

Comment:
I see MatrixSSL returns "not supported" error, but I didn't see note about this change in MatrixSSL changelog - maybe some DEFINE should be set to re-enable this feature support?

Point is, if this feature was removed - tests should be removed too, not commented out. If this feature is still there - maybe it should be re-enabled or some note added why it was disabled in our `Changes`.